libevmasm/LinkerObject.cpp

```diff
@@ -38,7 +38,7 @@ void LinkerObject::link(map<string, h160> const& _libraryAddresses)
 	std::map<size_t, std::string> remainingRefs;
 	for (auto const& linkRef: linkReferences)
 		if (h160 const* address = matchLibrary(linkRef.second, _libraryAddresses))
-			address->ref().copyTo(ref(bytecode).cropped(linkRef.first, 20));
```

Comment:
I noticed that this line is not tested. This has to be changed before this PR is merged.

Reply:
Actually it should have been tested, there are end-to-end tests that perform linking. Running locally to see.